FreeRTOS/FreeRTOS-Plus-TCP
# FreeRTOS_GetEMACTaskHandle function added

Add FreeRTOS_GetEMACTaskHandle Function for STM32H7xx Devices

Description
-----------
Added the FreeRTOS_GetEMACTaskHandle function for STM32H7xx devices. This function provides access to the EMACTaskHandle for managing the Ethernet MAC task within the FreeRTOS environment.

Test Steps
-----------
- Build the project with the new function integrated.
- Deploy the firmware to an STM32H7xx development board.
- Verify that the FreeRTOS_GetEMACTaskHandle function is correctly returning the EMACTaskHandle.
- Run any existing test cases that involve Ethernet MAC operations to ensure they work as expected.
- Validate that no regressions or issues have been introduced in other parts of the system.

Checklist:
----------

Related Issue
-----------
N/A

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
